src/FSharp.Formatting.ApiDocs/GenerateHtml.fs

```diff
@@ -618,10 +618,27 @@ type HtmlRender(model: ApiDocModel, ?menuTemplateFolder: string) =
         let isTemplatingAvailable =
             match menuTemplateFolder with
             | None -> false
-            | Some input -> FSharp.Formatting.Common.Menu.isTemplatingAvailable input
+            | Some input -> Menu.isTemplatingAvailable input
 
         if isTemplatingAvailable then
-            "TODO!"
+                 if otherDocs && model.Collection.CollectionName <> "FSharp.Core" then
```

Comment:
Please format your code before committing 😉.

Reply:
Sure ! Any formatter you recommend?

Reply:
I'd use the one they check in the CI, https://github.com/fsprojects/FSharp.Formatting/blob/main/.github/workflows/pull-requests.yml#L26